components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -214,6 +257,24 @@ export function CreateWorkspacePage() {
                         {isLoading ? "Loading ..." : isStarting ? "Creating Workspace ..." : "New Workspace"}
                     </Button>
                 </div>
+                {existingWorkspaces.length > 0 && (
+                    <div className="w-full flex flex-col justify-end px-6">
+                        <p className="mt-6 text-center text-base">Running workspaces on this revision</p>
```

Comment:
**thought**: I understand revision is probably more accurate, in case we could relax this a bit we could go with a friendlier and less scary term. Your call.

```suggestion
                        <p className="mt-6 text-center text-base">Running workspaces on this branch</p>
```

Reply:
This would not be correct. We are comparing the revision